pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -107,15 +117,24 @@ stdenv.mkDerivation rec {
     ninja
     autoconf # for vendored jemalloc
     flatbuffers
-  ] ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
+  ] ++ lib.optional enableCuda [
+    addOpenGLRunpath
```

Comment:
This should not be needed when using the hook. Or you want to patch paths that aren't checked by the hook?

Reply:
I am also patching the tests, which will run successfully with the right `--option extra-sandbox-paths` argument.

Reply:
Ah we should probably fix `addOpenGLRunpathHook` so that it runs on more things